app/src/main/java/ch/onepass/onepass/ui/organization/OrganizationDashboardScreen.kt

```diff
@@ -0,0 +1,327 @@
+            rating = organization.averageRating,
+            onClick = { onNavigateToProfile(organization.id) })
+
+        // Manage Events Section - Placeholder for PR4
+        Text(
+            text = "MANAGE EVENTS",
+            style = MaterialTheme.typography.titleMedium,
+            fontWeight = FontWeight.Bold,
+            color = MaterialTheme.colorScheme.onSurface,
+            modifier = Modifier.testTag(OrganizationDashboardTestTags.MANAGE_EVENTS_SECTION))
+
+        // Manage Staff Section - Placeholder for PR5
+        Text(
+            text = "MANAGE STAFF",
+            style = MaterialTheme.typography.titleMedium,
+            fontWeight = FontWeight.Bold,
+            color = MaterialTheme.colorScheme.onSurface,
+            modifier = Modifier.testTag(OrganizationDashboardTestTags.MANAGE_STAFF_SECTION))
+      }
+}
+
+/**
+ * Displays a summary card for the organization.
+ *
+ * @param organizationName The name of the organization.
+ * @param followers The number of followers.
+ * @param rating The average rating of the organization.
+ * @param onClick Callback invoked when the card is clicked.
+ */
+@Composable
+private fun OrganizationSummaryCard(
+    organizationName: String,
+    followers: Int,
+    rating: Float,
+    onClick: () -> Unit
+) {
+  Surface(
+      modifier =
+          Modifier.fillMaxWidth()
+              .shadow(elevation = 4.dp, shape = RoundedCornerShape(8.dp))
+              .clickable(onClick = onClick)
+              .testTag(OrganizationDashboardTestTags.ORG_SUMMARY_CARD),
+      shape = RoundedCornerShape(8.dp),
+      color = MaterialTheme.colorScheme.surfaceVariant) {
+        Row(
+            modifier = Modifier.fillMaxWidth().padding(16.dp),
+            verticalAlignment = Alignment.CenterVertically) {
+              // Placeholder for organization logo/image
+              Box(
+                  modifier =
+                      Modifier.size(72.dp)
+                          .background(
+                              MaterialTheme.colorScheme.surfaceContainer, RoundedCornerShape(6.dp)))
+
+              Spacer(modifier = Modifier.width(16.dp))
+
+              Column(modifier = Modifier.weight(1f)) {
+                Text(
+                    text = organizationName.uppercase(Locale.getDefault()),
+                    style = MaterialTheme.typography.titleLarge,
+                    fontWeight = FontWeight.Bold,
+                    color = MaterialTheme.colorScheme.onSurface,
+                    maxLines = 1,
+                    overflow = TextOverflow.Ellipsis,
+                    modifier = Modifier.testTag(OrganizationDashboardTestTags.ORG_NAME))
+
+                Spacer(modifier = Modifier.height(4.dp))
+
+                Row(
+                    horizontalArrangement = Arrangement.spacedBy(16.dp),
+                    verticalAlignment = Alignment.CenterVertically) {
+                      Text(
+                          text = "${followers / 1000f}K FOLLOWERS",
```

Comment:
Followers formatting can show “0.7K” for small orgs, 
`"${followers / 1000f}K"` will render decimals with default precision and also for values < 1000.
Fix: only abbreviate ≥1000 and format to one decimal using the following helper (that can later be improved to support larger numbers, as done in the `OrganizationCard`)  :

```suggestion
                          text = "${formatFollowers(followers)} FOLLOWERS",
```
Basic `fomatFollowers` :
```
private fun formatFollowers(n: Int): String =
  if (n >= 1000) String.format(Locale.US, "%.1fK", n / 1000f).replace(".0", "")
  else n.toString()
```